documentation/development-docs/design-guidelines/should-process-confirm-impact.md

```diff
@@ -89,18 +93,18 @@ void ConfirmAction(bool force, string continueMessage, string processMessage, st
 
 ## ConfirmImpact
 
-From [Cmdlet Attribute Declaration](https://msdn.microsoft.com/en-us/library/system.management.automation.confirmimpact(v=vs.85).aspx):
```

Comment:
I thought we don't encourage this anymore.

Reply:
@maddieclayton updated the wording in this section to discourage setting this property